src/renate/utils/misc.py

```diff
@@ -12,3 +16,24 @@ def int_or_str(x: str) -> Union[str, int]:
         return int(x)
     except ValueError:
         return x
+
+
+def possibly_populate_mask_and_kill_logits(
```

Comment:
can we rename the function? I would refrain from "martial language". does replacing "kill" with "ignore work for you?.

at different places we add the prefix "maybe" to indicate that the function is potentially executed. Maybe we can follow this convention? (possibly -> maybe)

Reply:
Changed.